### PR TITLE
improve module sizing options

### PR DESCRIPTION
**Pull request by nullobsi**
Add `min-length` config property, can be used in combination with `max-length` to fix the length of a module to a specific size
Also add `align` property to align text in a label

this should help with #883

### CODE REVIEW

File: src/ALabel.cpp [hunk 1]
```diff
@@ -20,14 +20,23 @@ ALabel::ALabel(const Json::Value& config, const std::string& name, const std::st
...
     label_.set_single_line_mode(true);
   }
 
+  if (config_["min-length"].isUInt()) {
+    label_.set_width_chars(config_["min-length"].asUInt());
+  }
+
+  if (config_["align"].isDouble()) {
+    auto align = config_["align"].asFloat();
+    label_.set_xalign(align);
```
Comment (Alexays) [inline src/ALabel.cpp:37]: Does we also want xalign if the bar is in vertical mode?

Comment (nullobsi) [reply src/ALabel.cpp:37]: I think it should be based on the mode of the bar. Let me try and add a check.

Comment (nullobsi) [reply src/ALabel.cpp]: Okay, I've made it based off the rotate property. This should make align match with how the max and min lengths affect the label.

**Comment by Alexays** [comment]
Nice, could you add `min-length` to man, then i'll merge this PR :)

**Comment by nullobsi** [comment]
I've updated the man pages that have `max-length`. Thank you!

**Comment by Alexays** [comment]
Thanks!

**Pull request merged**